### PR TITLE
document `extraResources` field in tenant helm chart

### PR DESCRIPTION
**Pull request by pjuarezd**

### CODE REVIEW

**Review by jiuker — APPROVED** [review]

**Review by shtripat — APPROVED** [review]
LGTM. Please check the shellcheck warning.

**Comment by pjuarezd** [comment]
@feorlen, we will wait for your review before merge this PR

File: helm/tenant/values.yaml
Comment (feorlen) [inline helm/tenant/values.yaml:285]:
```suggestion
## with the helm deployment.
```

**Review by feorlen — DISMISSED** [review]
👍🏻 
Suggested a small edit to the comment explaining what this is for

**Review by feorlen — APPROVED** [review]
👍🏻

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**
